### PR TITLE
Fixing the issues with audio.

### PR DESCRIPTION
**Pull request by mithro**

### CODE REVIEW

**Comment by mithro** [comment]
@thaytan Does this all look sane based on your changes from last night?

File: README.md [hunk 1]
```diff
@@ -256,7 +256,23 @@ gst-switch is tested with the following video frame rates;
...
 The default audio input port is *4000*.
 
-*FIXME: What is the required audio format?*
+Audio input must be in;
+```
+      audio/x-raw
+                 format: S16LE
+                   rate: 48000
+               channels: 2
+```
```
Comment (thaytan) [inline README.md]: This should also have "layout: interleaved"

File: README.md [hunk 2]
```diff
@@ -256,7 +256,23 @@ gst-switch is tested with the following video frame rates;
...
+
+##### Example test audio feed
+
+```
+gst-launch-1.0 audiotestsrc is-live=true ! audioconvert ! audio/x-raw,rate=48000,channels=2 ! gdppay ! tcpclientsink port=4000
+```
```
Comment (thaytan) [inline README.md]: For completeness, specify the capsfilter as 'audio/x-raw,rate=48000,channels=2,format=S16LE,layout=interleaved'

File: run-demo.sh [hunk 3]
```diff
@@ -2,12 +2,20 @@
...
+killall gst-switch-srv || true
+sleep 2
+./tools/gst-switch-srv -r &
+sleep 2
+./tools/gst-switch-ui &
+sleep 2
+
+gst-launch-1.0 audiotestsrc is-live=true \
+        ! audioconvert \
+        ! audio/x-raw,rate=48000,channels=2,layout=interleaved \
```
Comment (thaytan) [inline run-demo.sh]: Same here: audio/x-raw,rate=48000,channels=2,format=S16LE,layout=interleaved

File: README.md (hunk 1 above)
Comment (mithro) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (mithro) [inline README.md]: Done.

File: run-demo.sh (hunk 3 above)
Comment (mithro) [inline run-demo.sh]: Done.

**Pull request merged**